install/0000_90_machine-config-operator_01_prometheus-rules.yaml

```diff
@@ -67,7 +67,7 @@ spec:
       rules:
         - alert: MCDRebootError
           expr: |
-            max by(namespace, node, pod) (increase(mcd_reboots_failed_total[15m])) > 0
+            mcd_reboots_failed_total > 0
```

Comment:
with this change, I would be interested in adding a `for: 5min` to fire alert only when reboot attempt is failing after multiple attempts over 5min. @jan--f Does this makes sense?

Reply:
oh good thought, if we want to give it some slack to recover, this makes sense. we could also get a similar effect by changing the expression `max by(namespace, node, pod) (increase(mcd_reboots_failed_total[5m])) > 0`

Reply:
Yep that would work.

Reply:
Updated with a `for: 5min`  expression